base_limiter.go

```diff
@@ -58,3 +59,19 @@ func (l *BaseLimiter) isTargetExtensions(r *http.Request) bool {
 	}
 	return false
 }
+func validateKey(key string) error {
+	for _, k := range []string{"remote_addr", "host"} {
```

Comment:
remote_addr" and "host" may be const variables.

Reply:
I see.
I should to make constaiant variable host and remote_addr.

I will do afer cristmas..